packages/subscriber/src/components/add-form/index.tsx

```diff
@@ -49,12 +51,19 @@ export const AddSubscriberForm: FunctionComponent< Props > = ( props ) => {
 	/**
 	 * ↓ Fields
 	 */
+	const emailControlMaxNum = 10;
+	const emailControlPlaceholder = [
+		__( 'sibling@email.com' ),
```

Comment:
Just out of curiosity, why should we translate email addresses?

Reply:
The translator can localize words such as `sibling``, `parents`, and `friends`, which is just a hint for the end user.

Reply:
Got it. Thanks for your explanation.

Reply:
"email"  is also English. ;-)

Reply:
I think that we should use `@example.com`. `email.com` is a real URL and should not be used.

Reply:
Example.com would be better 👍  Moar PCYsg-eI0-p2

Reply:
I've updated the email domain part to `@example.com`